test/evergreen.yml

```diff
@@ -38,30 +38,33 @@ functions:
       remote_file: wiredtiger/${build_variant}/${revision}/artifacts/${dependent_task|compile}_${build_id}.tgz
       bucket: build_external
       extract_to: wiredtiger
-  "fetch artifacts from little-endian" :
+  "fetch artifacts from endian format" :
     - command: s3.get
       params:
         aws_key: ${aws_key}
         aws_secret: ${aws_secret}
-        remote_file: wiredtiger/little-endian/${revision}/artifacts/WT_TEST.tgz
+        remote_file: wiredtiger/${endian_format}/${revision}/artifacts/WT_TEST.tgz
         bucket: build_external
-        local_file: WT_TEST-little-endian.tgz
-    - command: archive.targz_extract
-      params:
-        path: "WT_TEST-little-endian.tgz"
-        destination: "wiredtiger/build_posix/test/format"
-  "fetch artifacts from big-endian" :
+        extract_to: wiredtiger/build_posix/test/format
```

Comment:
It looks like after switching to using `extract_to` we lose the capacity to set the tarball name (or `path` parameter) as in `archive.targz_extract`. Having the file name reflecting the endian format is useful, as we have tests to verify data files from both the same or a different endianness system.

Reply:
I made a suggestion below on line 1891.

Reply:
Yes. It would help with debugging with a spawn host.

Reply:
I did it in 5141f0800a9fabe9b0ee7c867a0969aff5d57242

Reply:
Thanks.